libfirm-rs/src/entity.rs

```diff
@@ -62,3 +65,18 @@ impl Entity {
         }
     }
 }
+
+#[allow(clippy::derive_hash_xor_eq)]
```

Comment:
this allow is not required? your implementing both by hand.

Reply:
Fixed.